site/source/tutorials/008_animations/index.md

```diff
@@ -0,0 +1,370 @@
+---
+layout: tutorial
+title: Web Animations
+overview: Use the Dojo 2 WebAnimations meta to animate your widgets.
+paginate: true
+---
+
+{% section 'first' %}
+
+# Web Animations
+
+## Overview
+This tutorial introduces two lovable zombies which we will learn to animate using the Dojo 2 `WebAnimation` meta. We will introduce the api provided by WebAnimations and show you how to use it with Dojo 2.
+
+![Zombies](./zombieCover.png)
+
+## Prerequisites
+You can [download](../assets/008_animations-initial.zip) the initial demo project and run `npm install` to get started.
+
+The `@dojo/cli` command line tool should be installed globally. Refer to the [Dojo 2 local installation](../000_local_installation/) article for more information.
+
+You also need to be familiar with TypeScript as Dojo 2 uses it extensively. For more information, refer to the [TypeScript and Dojo 2](../../docs/fundamentals/typescript_and_dojo_2/) article.
+
+{% section %}
+
+## What is the web animations API
+
+The [Web Animations API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API) provides programatic control over web animations via the timing model and the animation model. This allows animations to be created and controlled via javascript with access to playbackrate, iterations, events etc... something which would previously have required the use of `requestAnimationFrame` or the dreaded `setInterval`. Dojo 2 provides a meta that can apply Web Animations to the rendered vdom in the widgets you create. This allows properties such as `play` and `duration` to be reactive to state changes and fits in neatly within the Dojo 2 eco stystem.
+
+## Introducing the zombies
+
+Looking at the `Zombies.ts` file within the initial project we can see that we have a series of divs representing two zombies, each with a body and two legs. Via the magic of css these turn into two zombies when you open your browser. To demonstrate the use of Web Animations with Dojo 2 we make the zombies walk towards each other when they are clicked.
```

Comment:
Looking at the `Zombies.ts` file within the initial project we can see that we have a series of divs representing two zombies -> Looking at `Zombies.ts` within the initial project, there are a series of `DIV`s representing two zombies